packages/pluginutils/src/extractAssignedNames.ts

```diff
@@ -1,27 +1,35 @@
+import type {
+  ArrayPattern,
+  AssignmentPattern,
+  Identifier,
+  ObjectPattern,
+  RestElement
+} from 'estree';
+
 import { ExtractAssignedNames } from '../types';
 
 interface Extractors {
   [key: string]: (names: string[], param: any) => void;
 }
 
 const extractors: Extractors = {
-  ArrayPattern(names: string[], param: import('estree').ArrayPattern) {
+  ArrayPattern(names: string[], param: ArrayPattern) {
```

Comment:
The `import()` style is used here to avoid confusion between extractors and estree. Can we leave this spot alone and file an issue with the linter?

Reply:
ok, I'll close this PR then